LondonBritishLibrary/orient/BLorient12125.xml

```diff
@@ -0,0 +1,229 @@
+                                <date notBefore="1800" notAfter="1978"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1v"/>
+                                    <title ref="LIT6925EnumerationOTAm"/>
+                                    <note>39 in number</note>
+                                    <q xml:lang="am">የብሉይ፡ ኪዳን፡ መጻሕፍት፡ እሊህ፡ ናቸው።<gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#229r"/>
+                                    <desc type="StampExlibris">The seal of Empress <persName ref="PRS9345taytuBe"/>.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#X"/>
+                                    <desc type="AcquisitionNote">A note on the verso of the first end flyleaf: "Bought of 
+                                        <persName ref="PRS14316Alkson">C. R. Alkson</persName>, Esq., Brentwood, Essex".</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#I"/>
+                                    <desc type="Unclear">Scribbles.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1" type="Boards"/>    
+                                <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                                <decoNote xml:id="b4" type="Spine">Leather spine repaired in the British Library.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1800" notAfter="1978">18th or 19th century</origDate>
+                        </origin>                        
+                        <provenance>According to a note on <locus target="#X"/> bought from a person with the name 
+                            <persName ref="PRS14316Alkson"/>.</provenance>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary "/>
```

Comment:
NB: also my mistake I did not notice: the blank space in target has caused significant instability in the application, as the bibliography could not be created! https://github.com/BetaMasaheft/Manuscripts/commit/4ca7ade65d9ff291fe78380e876e4404e0898c30

Reply:
this was one of the reasons behind the latest crashes

Reply:
Sorry for that! I will pay more attention in the future!